com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -409,7 +409,8 @@ internal void InitializeVariables()
                     instance.SetNetworkBehaviour(this);
 
                     var instanceNameProperty = fieldType.GetProperty(nameof(INetworkVariable.Name));
-                    instanceNameProperty?.SetValue(instance, sortedFields[i].Name);
+                    var sanitizedVariableName = sortedFields[i].Name.Replace("<", string.Empty).Replace(">k__BackingField", string.Empty);
+                    instanceNameProperty?.SetValue(instance, sanitizedVariableName);
```

Comment:
Will users notice this change or are these names only used internally?

Reply:
It's only meant to be used internally, but the getter is public.

If anything, users will notice an improvement :) But they've never seen the way we did it before anyways.